docs/user/dashboard/dashboard.asciidoc

```diff
@@ -0,0 +1,472 @@
+
+To change the panel type, click on the panel options: 
+
+[role="screenshot"]
+image::images/tsvb_change_display.gif[TSVB change the panel type]
+
+[float]
+[[custommize-the-data]]
+==== Customize the data
+
+View data in a different <<index-patterns,index pattern>>, and change the data label name and colors. The options available depend on the panel type. 
+
+To change the index pattern, click *Panel options*, then enter the new *Index Pattern*.
+
+To override the index pattern for a data set, click *Data > Options*. Select *Yes* to override, then enter the new *Index pattern*.
+
+To change the data labels and colors:
+
+. Click *Data*.
+. Enter the *Label* name, which *TSVB* uses on the legends and data labels.
+. Click the color picker, then select the color for the data.
++
+[role="screenshot"]
+image::images/tsvb_color_picker.png[TSVB color picker]
+
+[float]
+[[add-annotations]]
+==== Add annotations
+
+You can overlay annotation events on top of your *Time Series* charts. The options available depend on the data source.
+
+To begin, click *Annotations*, click *Add data source*, then configure the options.
+
+[role="screenshot"]
+image::images/tsvb_annotations.png[TSVB annotations]
+
+[float]
+[[filter-the-panel]]
+==== Filter the panel
+
+The data that displays on the panel is based on the <<index-patterns,index pattern>> and <<set-time-filter,time filter>>. 
+You can filter the data on the panels using the <<lucene-query,Lucene query syntax>>.
+
+Click *Panel options*, then enter the syntax in the *Panel Filter* field.
+
+If you want to ignore filters from all of {kib}, select *Yes* for *Ignore global filter*.
+
+[float]
+[[vega]]
+=== Create custom panels with Vega
+
+Build custom visualizations using *Vega* and *Vega-Lite*, backed by one or more data sources including {es}, Elastic Map Service, 
+URL, or static data. Use the {kib} extensions to embed *Vega* in your dashboard, and add interactive tools.
+
+Use *Vega* and *Vega-Lite* when you want to create a visualization for:
+
+* Aggregations that use `nested` or `parent/child` mapping
+* Aggregations without an index pattern
+* Queries that use custom time filters
+* Complex calculations
+* Extracting data from _source instead of aggregations
+* Scatter charts, sankey charts, and custom maps
+* Using an unsupported visual theme
+
+[role="screenshot"]
+image::images/vega.png[Vega UI]
+
+*Vega* and *Vega-Lite*  are declarative formats that:
+
+* Create complex visualizations
+* Use JSON and a different syntax for declaring visualizations
+* Are not fully interchangeable
+
+For more information about *Vega* and *Vega-Lite*, refer to:
+
+* <<vega-lite-tutorial-create-your-first-visualizations,Create your first visualization>>
+* <<vega-tutorial-update-kibana-filters-from-vega,Update Kibana filters>>
+* <<reference-for-kibana-extensions,Reference for Kibana extensions>>
+* <<resources-and-examples,Resources and examples>>
+
+[float]
+[[timelion]]
+=== Create panels with Timelion
```

Comment:
I thought we already made the decision to deprecate Timelion. It surprises me that it's listed here

Reply:
We deprecated the Timelion app, but not the Timelion editor.